README.md

```diff
@@ -44,7 +46,7 @@ The linter can be launched using:
 dart analyze
 ```
 
-Dart allows you to fix issues in your code with the `dart command`.
+Dart allows you to fix issues in your code with the dart command `dart fix`.
```

Comment:
You are missing the `--apply` to apply the fixes

Reply:
Read the lines below, I show both options `apply` and `dry-run`

Reply:
Ok, that's my bad